static/src/javascripts/projects/common/modules/experiments/tests/hosted-autoplay.js

```diff
@@ -13,26 +23,55 @@ define([], function () {
         this.idealOutcome = 'People will either more often click on the next hosted video or wait until end of the current video to be redirected into the next video page url.';
 
         this.canRun = function () {
-            return true;
+            return config.page.contentType === 'Video' && config.page.tones === 'Hosted';
         };
 
         this.variants = [
             {
                 id: 'control',
-                test: function () {
-
+                test: function () {},
+                success: function (complete) {
+                    bean.on(qwery('.hosted__next-video--tile')[0], 'click', function (){
+                        complete();
+                    });
+                    bean.on(qwery('.hosted__container--full')[0], 'click', function (){
+                        complete();
+                    });
                 }
             },
             {
                 id: 'variant1',
-                test: function () {
-
+                test: function () {},
+                success: function (complete) {
+                    bean.on(qwery('.hosted__next-video--tile')[0], 'click', function (){
+                        complete();
+                    });
+                    bean.on(qwery('.hosted__container--full')[0], 'click', function (){
+                        complete();
+                    });
+                    bean.on(qwery('.hosted-next-autoplay__poster')[0], 'click', function (){
+                        complete();
+                    });
+                    mediator.on('hosted video: autoredirect', complete);
                 }
             },
             {
                 id: 'variant2',
-                test: function () {
-
+                test: function () {},
+                success: function (complete) {
+                    bean.on(qwery('.hosted__next-video--tile')[0], 'click', function (){
```

Comment:
Just to avoid some noise here, I think it would be worth changing these calls to `bean.on(qwery('.hosted__next-video--tile')[0], 'click', complete)`